client/keys.go

```diff
@@ -36,6 +36,7 @@ const (
 	ErrorCodeNodeExist   = 105
 	ErrorCodeRootROnly   = 107
 	ErrorCodeDirNotEmpty = 108
+	EcodeUnauthorized    = 110
```

Comment:
hm... We have inconsistent const for server and client. I will follow the client side anyway.